docs/commands/translate.md

```diff
@@ -19,12 +19,13 @@ redocly translate --version
 
 ## Options
 
-| Option          | Type    | Description                                                                                                                            |
-| --------------- | ------- | -------------------------------------------------------------------------------------------------------------------------------------- |
-| `locale`        | string  | **REQUIRED** Name of a locale folder, inside your project's `l10n` directory, to generate translations for. Use `all` for all locales. |
-| `--lint-config` | string  | Severity level for config file linting. Possible values: `warn`, `error`, `off`. Defaults to `warn`.                                   |
-| `--help`        | boolean | Show help.                                                                                                                             |
-| `--version`     | boolean | Show version number.                                                                                                                   |
+| Option                | Type    | Description                                                                                                                            |
+| --------------------- | ------- | -------------------------------------------------------------------------------------------------------------------------------------- |
+| `locale`              | string  | **REQUIRED** Name of a locale folder, inside your project's `l10n` directory, to generate translations for. Use `all` for all locales. |
+| `--lint-config`       | string  | Severity level for config file linting. Possible values: `warn`, `error`, `off`. Defaults to `warn`.                                   |
+| `--project-dir`, `-d` | string  | Specifies the project folder to execute `translate` command in. Used for managing multiple projects.                                   |
```

Comment:
Can we use the same explanation as we use for the other commands? Where `project-dir` is the directory where the Redocly project source is?

Reply:
Updated to match the wording from `preview`. Will do the same for `eject`.